cfg/conf.sample.php

```diff
@@ -56,6 +56,12 @@
 ; the pastes encryption key
 ; urlshortener = "https://shortener.example.com/api?link="
 
+; you can set urlshortenerajax to yes if the shortener returns the url plain text
```

Comment:
maybe rather "true" than "yes"

Reply:
also explain that you should point the `urlshortener` to some API endpoint that does this, maybe also link to YOURLS doc for that as we mention it above as an example

Reply:
personaly i think "plain text" should be enough, not every body is using YOURLS, i for my self using a self written one.